R/model.R

```diff
@@ -1,15 +1,54 @@
 #' @title Run the simulation
 #' @description
-#' The main entrypoint for the simulation. run_simulation puts together the
-#' model components and runs the malaria simulation. This currently returns a
-#' dataframe with the number of individuals in each state at each timestep
+#' Run the simulation for some time given some parameters.
 #'
-#' Warning: the columns of the output dataframe is likely to change as we figure
-#' out what kind of outputs we would like to report from the simulation.
+#' The resulting dataframe contains the following columns:
 #'
-#' @param timesteps the number of timesteps to run the simulation for
+#' timestep: the timestep for the row
+#' infectivity: the infectivity from humans towards mosquitoes
```

Comment:
"the probability of transmitting infection from humans to mosquitoes with each infectious bite"?

Reply:
hmm, it's not really a probability. It's a model specific value